docs/devs/e2e-tests.md

```diff
@@ -0,0 +1,58 @@
+# E2E Tests
+
+## Introduction
+    E2E testing is a critical aspect of Ozone HIS that ensures all the workflows work as expected.
+
+## Prerequisites
+    Before running Ozone E2E tests, ensure you have Node.js and npm installed on your machine.
+
+## Setup Instructions
+    Clone Ozone E2E repository: git clone https://github.com/ozone-his/ozone-e2e.
+    Navigate to the project directory: cd ozone-e2e.
+    Install dependencies: yarn install.
```

Comment:
The command for installing dependencies is yarn install, but yarn is not listed in the prerequisites. If yarn is a prerequisite, it should be listed. If not, the command should be `npm install`.